login.py

```diff
@@ -48,16 +53,16 @@ def login(address: str,username: str,password: str, ignore_cert=False):
     :param bool,optional ignore_cert: Ignore and disable certificate warnings
     """
     if ignore_cert is True:
-        config.http['ssl'] = 'Trust'
+        http.config.http['ssl'] = 'Trust'
```

Comment:
Not sure this is what you want to set.
I think you should import the config and give it a name: `from cterasdk import config as cterasdk_config` and then here you set it like this: `cterasdk_config.http['ssl'] = 'Trust'`